openfisca_core/entities.py

```diff
@@ -297,9 +309,30 @@ def members_position(self):
 
         return self._members_position
 
+    @property
+    def members_entity_id(self):
```

Comment:
> Not sure about the interest of using a property if we don't have any logic in the getter and setter. Is it just for consistency with the other attributes?

Reply:
@fpagnoux Consistency, and easier debugging later. Using a property makes it easier to detect who is accessing this attribute. It makes sense to encapsulate this attribute somewhat "eagerly" because it's clearly an implementation detail that has its own structure and must be kept in sync with others, not something it's natural to expose to the world (like e.g. a `name`).

Reply:
> 👌